### ISSUE TITLE
Tokenizer error on double quoted import

### ISSUE THREAD

**Issue by saitho** [martin-helmich/typo3-typoscript-lint#89]
TypoScript imports with double quotes will lead to a tokenizer error.

```
=> /home/saitho/Workspace/typo3ext/Configuration/TypoScript/setup.typoscript.
  39 Tokenization error: Cannot tokenize line "@import "EXT:typo3ext/Configuration/TypoScript/Setup/Module/module.typo3ext.typoscript""
```

Not sure what TYPO3 guidelines say about using single or double quotes here.
Technically both work, so the linter should not throw an error here.
If guidelines say one should use single quotes, it could output a proper hint for that. :)

**Comment by saitho** [comment]
Just noticed this is an issue for the parser. Opened a PR there: https://github.com/martin-helmich/typo3-typoscript-parser/pull/46

**Comment by martin-helmich** [comment]
Fixed via martin-helmich/typo3-typoscript-parser#46
